sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -112,7 +112,7 @@ az appconfig credential list --name <config-store-name>
 
 Alternatively, get the connection string from the Azure Portal.
 
-#### Create Client
+#### Create a configuration client
```

Comment:
I would revert all of these casing changes and go back to "Create a Configuration Client"